config/clients/java/template/Java/libraries/native/build.gradle.mustache

```diff
@@ -0,0 +1,116 @@
+plugins {
+    id 'java'
+    id 'maven-publish'
+    id 'idea'
+    id 'eclipse'
+    id 'com.diffplug.spotless' version '6.+'
+}
+
+group = '{{groupId}}'
+version = '{{artifactVersion}}'
+
+repositories {
+    mavenCentral()
+}
+
+sourceCompatibility = JavaVersion.VERSION_11
+targetCompatibility = JavaVersion.VERSION_11
+
+// Some text from the schema is copy pasted into the source files as UTF-8
+// but the default still seems to be to use platform encoding
+tasks.withType(JavaCompile) {
+    configure(options) {
+        options.encoding = 'UTF-8'
+    }
+}
+javadoc {
+    options.encoding = 'UTF-8'
+}
```

Comment:
This is only required for supporting older versions of Eclipse editors.

Refer to: https://github.com/gradle/gradle/issues/898

```suggestion
```